.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: python
+python:
+  - 3.4
+  - 3.5
```

Comment:
oi! The Travis builds are faster than Circle CI :P
But, no, Travis runs the jobs in parallel, so adding more jobs doesnt impact elapsed time until there are more than 5 concurrent jobs, and I have seen up to 15 concurrent OSS jobs in Travis off-peak times.